src/components/Users/UserAvailabilityTab.tsx

```diff
@@ -199,8 +199,8 @@ export default function UserAvailabilityTab({ userData: user }: Props) {
                                 <p className="text-sm text-gray-600">
                                   {Math.floor(
                                     getSlotsPerSession(
-                                      availability[0].start_time,
-                                      availability[0].end_time,
+                                      availability[0]?.start_time ?? "00:00",
+                                      availability[0]?.end_time ?? "23:59",
```

Comment:
let's not assume time values. the right approach here is, check if availability[0] exists and then only call the Math.floor(getSlotsPerSession...)